src/main/java/seedu/address/logic/parser/ParserType.java

```diff
@@ -15,16 +21,11 @@ public static CommandParser get(String commandWord){
 				}
 			} 
 			catch (NoSuchFieldException e) {
-				return new IncorrectCommandParser();
-			} catch (SecurityException e) {
-				// TODO Auto-generated catch block
-				e.printStackTrace();
-			} catch (InstantiationException e) {
-				// TODO Auto-generated catch block
-				e.printStackTrace();
-			} catch (IllegalAccessException e) {
-				// TODO Auto-generated catch block
-				e.printStackTrace();
+				logger.warning("Error: Non-parser class placed into list");
+				System.exit(0);
```

Comment:
Okay not sure if we should exit here or pass the problem to some other class to handle - but anyways it looks good now, will start integration